prow/config.yaml

```diff
@@ -1556,6 +1556,52 @@ presubmits:
     context: pull-kubernetes-verify
     rerun_command: "/test pull-kubernetes-verify"
     trigger: "(?m)^/test( all| pull-kubernetes-verify),?(\\s+|$)"
+  - name: pull-kubernetes-verify-prow
+    agent: kubernetes
+    always_run: false
+    context: pull-kubernetes-verify-prow
+    rerun_command: "/test pull-kubernetes-verify-prow"
+    trigger: "(?m)^/test pull-kubernetes-verify-prow,?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/prow-test-image:1.9-latest
+        imagePullPolicy: Always
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        # docker-in-docker needs privilged mode
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: cache-ssd
+          mountPath: /root/.cache
+        - name: docker-graph
+          mountPath: /docker-graph
+        - name: var-lib-docker
+          mountPath: /var/lib/docker
+        ports:
+        - containerPort: 9999
+          hostPort: 9999
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+      - name: docker-graph
+        hostPath:
+          path: /mt/disks/ssd0/docker-graph
+      - name: var-lib-docker
+        emptyDir: {}
```

Comment:
what's this `emptyDir`? Probably you need to add it to https://github.com/kubernetes/test-infra/blob/master/prow/kube/types.go#L77-L83

Reply:
https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

Reply:
we might not need this eventually, but I don't know if our hosts use for the docker images and the path this is mounted to needs to me something else, so this volume is used to map in a tempdir to /var/lib/docker

Reply:
I mean you need to add it to prow/kube, or prow will not understand it :-)

Reply:
https://github.com/kubernetes/test-infra/pull/5343

Reply:
merged, everything redeployed